airbyte-integrations/connectors/destination-postgres/src/main/java/io/airbyte/integrations/destination/postgres/PostgresCli.java

```diff
@@ -0,0 +1,55 @@
+package io.airbyte.integrations.destination.postgres;
+
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.db.jdbc.JdbcDatabase;
+import io.airbyte.db.jdbc.JdbcUtils;
+import io.airbyte.integrations.base.ssh.SshTunnel;
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang3.tuple.Pair;
+
+public class PostgresCli {
+
+  public static void main(final String[] args) throws Exception {
+    if (args.length < 3 || args.length % 2 != 1) {
+      System.out.println("Usage: java -jar whatever.jar <postgres-config> <schema1> <table1> [<schema2> <table2> ...]");
+      System.exit(1);
+    }
```

Comment:
Not gonna nitpick too much but just want to make sure I understand the logic here
argument 0 - config
then all arguments should be in pairs, with at least 3 args (config, schema, table)
so it should be >= 3 and if >3 an odd number?
Logic is fine just making sure I'm reading that correctly

Reply:
yep, that's exactly the intent